Applications/SimpleShot/task_runner.cpp

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file	task_runner.cpp
+ * @date	08 Jan 2021
+ * @brief	task runner for the simpleshot demonstration
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <unistd.h>
+
+#include <app_context.h>
+#include <model.h>
+
+#include "layers/centering.h"
+#include "layers/centroid_nearest_neighbors.h"
+#include "layers/l2norm.h"
+
+namespace simpleshot {
+
+namespace {
+unsigned int getNumClass() { return 5; }
+
```

Comment:
Need to get input or parameter. not hard coded.

Reply:
Yes, this will be changed.

Reply:
Done